src/effects/VST3/VST3Wrapper.cpp

```diff
@@ -109,8 +111,120 @@ bool SetupProcessing(Steinberg::Vst::IComponent& component, Steinberg::Vst::Proc
    return false;
 }
 
+class InputParameterChanges final : public Steinberg::Vst::IParameterChanges
+{
+   const Steinberg::int32 mParameterCount;
+   const SingleInputParameterValue* mParameterQueues;
```

Comment:
Could this instead be `SingleInputParameterValue *const mParameterQueues;` so you can remove the `const_cast` below?